cli/cmd/uninstall/uninstall.go

```diff
@@ -117,125 +123,112 @@ func (c *Command) init() {
 
 func (c *Command) Run(args []string) int {
 	c.once.Do(c.init)
-
-	// The logger is initialized in main with the name cli. Here, we reset the name to uninstall so log lines would be prefixed with uninstall.
 	c.Log.ResetNamed("uninstall")
-
-	defer func() {
-		if err := c.Close(); err != nil {
-			c.Log.Error(err.Error())
-			os.Exit(1)
-		}
-	}()
+	defer common.CloseWithError(c.BaseCommand)
 
 	if c.helmActionsRunner == nil {
 		c.helmActionsRunner = &helm.ActionRunner{}
 	}
 
 	if err := c.set.Parse(args); err != nil {
-		c.UI.Output(err.Error(), terminal.WithErrorStyle())
-		return 1
-	}
-	if len(c.set.Args()) > 0 {
-		c.UI.Output("Should have no non-flag arguments.", terminal.WithErrorStyle())
```

Comment:
if you are changing this here, you will also need to know that the other commands have similar output messages.  so we either need to change them as well or remove this from the PR.  I think any UX messages we probably want to be more explicit and forward thinking where a designer / product person is helping dictate these changes so that they are consistent and done in isolation.  Otherwise, each of us engineers will just refactor messages here and there and make our already inconsistent UX more inconsistent.  IMO, I would remove most of the discretionary changes you have in here related to output messages that are not related to the deleting of CRs/CRDs and make a separate PR.